packages/extension-chains/src/bundle.ts

```diff
@@ -10,15 +10,7 @@ import { base64Decode } from '@polkadot/util-crypto';
 
 export { packageInfo } from './packageInfo';
 
-const expanded = new Map<string, Chain>();
```

Comment:
This is actually not much of a cache after moving to the volatile service workers, but it does make tests more painful, so I deleted it. Checked that there doesn't seem to be any caching-worthy logic in here anyway.